swap/Cargo.toml

```diff
@@ -31,6 +31,7 @@ libp2p-async-await = { git = "https://github.com/comit-network/rust-libp2p-async
 miniscript = { version = "5", features = ["serde"] }
 monero = { version = "0.10", features = ["serde_support"] }
 monero-rpc = { path = "../monero-rpc" }
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Explicitly adding a transitive dependency might cause trouble down the line when we want to update them because we need to match the versions.

Which dependency brings in openssl? You can use `cargo tree` to figure that out.
Can we either toggle vendoring through a feature flag on that dependency or switch to a rust-based TLS implementation to avoid this?